release-calendar/src/client/stylesheets/app.scss

```diff
@@ -0,0 +1,64 @@
+.main-container {
+    padding: 12px;
+    padding-top: 24px;
+}
+
+@mixin container-shadow {
+    box-shadow:
+        rgba(0, 0, 0, 0.2) 0px 2px 4px -1px,
+        rgba(0, 0, 0, 0.14) 0px 4px 5px 0px,
+        rgba(0, 0, 0, 0.12) 0px 1px 10px 0px;
+    padding: 16px;
+    border-radius: 4px;
+}
+
+.col-side-pannel {
+    @include container-shadow;
+    padding-top: 0px;
+    width: 30%;
+    float:left;
+}
+
+.col-calendar {
+    @include container-shadow;
+    width: 63%;
+    float:right;
+}
+
+@media only screen and (max-width: 940px) {
+    .col-side-pannel {
+        float:left;
+        width: 50%;
+    }
+    .col-calendar {
+        width: 90%;
+        margin-top: 5px;
+        clear: both;
+        float: left;
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    .col-side-pannel {
+        width: 70%;
+    }
+}
+
+body {
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
+    font-size: 14px;
+    line-height: 3em;
+    color: #24292e;
+    margin: 0px;
+}
+
+button {
+    -webkit-appearance: none;
+    font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji;
+    font-size: 14px;
+    line-height: 3em;
```

Comment:
This is the same as body styles. Could be removed.

Reply:
Buttons dont inherit by default. I did change things for readability to
`font-family: inherit;
    font-size: inherit;`